packages/mjml-text/README.md

```diff
@@ -1,16 +1,17 @@
 ## mj-text
 
-This tag allows you to display text in your email.
+This tag allows you to display text and HTML in your email and has shorthand attributes for easily styling its children.
```

Comment:
> and has shorthand attributes for easily styling its children

Not sure this provide any value here. But 👍 to add HTML mentioned

Reply:
not sure what is the purpose of `font-family="Helvetica" color="#F45E43"` etc other than being shorthands

Reply:
I think this is self-explanatory, as you can say this for any component